Add changelog for 3.15.0 release (#4616)
alphagov/govuk-frontend: CHANGELOG.md

# Changelog

## 3.15.0 (Feature release)

### New features

#### Update to the new GOV.UK logo

> The GOV.UK logo has been updated to reflect the changing of the monarch; King Charles III uses the Tudor Crown, rather than the St Edward’s Crown chosen by Queen Elizabeth II.

Author: N.B. This line is lifted almost verbatim from the crown comms document.

# Changelog

## 3.15.0 (Feature release)

### New features

#### Update to the new GOV.UK logo

The GOV.UK logo has been updated to reflect the changing of the monarch; King Charles III uses the Tudor Crown, rather than the St Edward’s Crown chosen by Queen Elizabeth II.

If your service uses GOV.UK branding, you should update your service to use the new crown as soon as possible.

These changes were made in the following pull requests:

- [#4379: Implement the Tudor crown favicons (v3.x)](https://github.com/alphagov/govuk-frontend/pull/4379)
- [#4380: Implement the Tudor crown in the Header component (v3.x)](https://github.com/alphagov/govuk-frontend/pull/4380)

##### Include the new logo assets

> Multiple new image assets are included in this release. These will need to be copied to your service's image assets folder if they aren't being used directly from the Frontend package. By default this folder is located at `/assets/images`.

Reviewer: ```suggestion
Multiple new image assets are included in this release. You will need to copy these to your service's image assets folder if they aren't being used directly from the Frontend package. By default this folder is located at `/assets/images`.
```

If you are using Nunjucks, the asset path may have been changed by the `assetPath` global variable or `assetsPath` parameter on the header component.

Copy the following files from `/dist/assets/images` into your assets folder. Any images with the same name as an existing image can be safely overwritten.

- favicon.ico
- govuk-apple-touch-icon-152x152.png
- govuk-apple-touch-icon-167x167.png
- govuk-apple-touch-icon-180x180.png
- govuk-apple-touch-icon.png
- govuk-logotype-tudor-crown.png
- govuk-mask-icon.svg
- govuk-opengraph-image.png

##### Update the logo in the header of your page

If you are using the `govukHeader` Nunjucks macro in your service, add the `useTudorCrown` parameter to the macro instantiation.

```nunjucks
{{ govukHeader({
  ...
  useTudorCrown: true
}) }}
```

> If you aren't using the Nunjucks macro, locate the HTML for the existing crown and replace it with this updated HTML. Make sure that the URL for the new PNG fallback image is correct.

Reviewer: ```suggestion
If you are not using the Nunjucks macro, locate the HTML for the existing crown and replace it with this updated HTML. Make sure the URL for the new PNG fallback image is correct.
```
